build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -266,5 +269,25 @@ private static void FixPermissions(string directory)
                 FS.FixModeFlags(directory);
             }
         }
+
+        private static string GetSharedFrameworkPublishPath()
```

Comment:
Yes, we should get this in for 4/14 and any new infra would rebase against this to pick it up.